WordPress/Classes/Services/BlogService.h

```diff
@@ -50,18 +50,57 @@ extern NSString *const WordPressMinimumVersion;
  */
 - (nullable Blog *)primaryBlog;
 
+/**
+ *  Sync all available blogs for an acccount
+ *
+ *  @param account the account for the associated blogs.
+ *  @param success a block that is invoked when the sync is sucessfull.
```

Comment:
❤️ all these doc comments! But you need one extra 'c' and one less 'l' in every 'successful'.